cmd/flow-dps-server/main.go

```diff
@@ -83,6 +85,14 @@ func run() int {
 	}
 	defer db.Close()
 
+	// Initialize storage library.
+	codec, err := zbor.NewCodec()
+	if err != nil {
+		log.Error().Err(err).Msg("could not initialize storage codec")
+		return failure
+	}
+	storage := storage.NewLibrary(codec)
```

Comment:
If we use `storage.NewLibrary`, imo the interface should also be called `Library`. `storage` is the package, so I wouldn't use that as the package name. Otherwise, you could simply do `storage.New(codec)`.